build_tools/travis/install.sh

```diff
@@ -53,11 +53,14 @@ if [[ "$DISTRIB" == "conda" ]]; then
     if [[ "$INSTALL_MKL" == "true" ]]; then
         conda create -n testenv --yes python=$PYTHON_VERSION pip nose \
             numpy=$NUMPY_VERSION scipy=$SCIPY_VERSION numpy scipy \
-            cython=$CYTHON_VERSION libgfortran mkl flake8
+            cython=$CYTHON_VERSION libgfortran mkl flake8 \
+            ${PANDAS_VERSION+pandas=$PANDAS_VERSION}
```

Comment:
creating an env with Pandas is conditional on $PANDAS_VERSION being set, as it's currently only in the 3.5 latest test image. Thus, this checks to see whether it is set and if it is, adds it to the env. see https://github.com/scikit-learn/scikit-learn/pull/7483#discussion_r80371773